docs/minio-client-complete-guide.md

```diff
@@ -798,6 +802,15 @@ mc policy download play/mybucket/myphotos/2020/
 Access permission for ‘play/mybucket/myphotos/2020/’ is set to 'download'
 ```
 
+*Example : Set anonymous bucket policy from a JSON file*
+
+Configure bucket policy for ``mybucket/myphotos/2020/`` sub-directory with a policy JSON file.
+
+```sh
+mc policy /tmp/policy.json play/mybucket/myphotos/2020/
```

Comment:
I didn't test this yet but I am skeptical that this works as advertised here. AFAIK, bucket polices are associated to buckets and not to `bucket/prefix` and if the user wants to set a custom policy for a bucket/prefix, then this needs to be specified in the JSON file itself, what do you think @Praveenrajmani ?

Reply:
Yes @vadmeste , you are right. Will change this